### ISSUE TITLE
⭐️ Feature: Implement Reply functionality in projects comment section 

### ISSUE THREAD

**Issue by theslowdeveloper** [CircuitVerse/CircuitVerse#4205]
### Is your feature request related to a problem? Please describe

Currently there is no functionality of replying in projects comment section
![Screenshot from 2023-10-20 10-55-56](https://github.com/CircuitVerse/CircuitVerse/assets/102225113/5a83eb7a-25be-4140-90e2-e8609295048b)

### Describe the solution you'd like

I think there should be reply button beneath each comment and users can click it to reply to a specific user in a similar comment like section.

### Are you working on this?

Yes

**Comment by test1883** [comment]
@theslowdeveloper hey can I work on this?

**Comment by tanmoysrt** [comment]
cc @tachyons @vedant-jain03

**Comment by aialok** [comment]
Can I work on this issue

**Comment by Fareed95** [comment]
is this issue is still open.. if then can you please give me a chance to resolve it .. please

**Comment by Ansh101112** [comment]
Assign it to me i will work on it.

**Comment by fillingtothemomo** [comment]
Hello, is this issue verified. If yes I would like to work on it.

**Comment by Himanshu-kiran** [comment]
Yes, this is important one.
Can you please confirm @tachyons, so that I could start working on this.

**Comment by Himanshu-kiran** [comment]
Update:
![Image](https://github.com/user-attachments/assets/c5580bec-b61c-4876-8d27-9a4a2d2cdf90)
Posting reply comment attaches comment to top level but not in nested.So, working on that.
Welcome any suggestion!

**Comment by vedant-jain03** [comment]
@salmoneatenbybear Approved ✅

**Comment by salmoneatenbybear** [comment]
@senbo1 would you still be interested in working on this ?

**Comment by senbo1** [comment]
yeah, I am opening a PR for this, you can collaborate if you want
